tests/integration_tests/functional/test_cpu_features_host_vs_guest.py

```diff
@@ -70,6 +71,91 @@
     "xtpr",
 }
 
+INTEL_GUEST_ONLY_FEATS = {
```

Comment:
commit message: what is the specific problem in https://github.com/firecracker-microvm/firecracker/commit/f4f7536eae69dce49895b476b0e58fa8fc597b7d that this fixes?

Reply:
Ouch. Yeah the commit started as a fix, but was fixed in 761909de71d2e4a8c110334cde928fab34831f61 instead. This now fleshes out the test.

Reply:
Fixed.